lib/board.rb

```diff
@@ -0,0 +1,63 @@
+require 'pry'
+class Board
+  attr_accessor :cells
+  def initialize
+    @cells = [" "," "," "," "," "," "," "," "," "]
+  end
+
+  def reset!
+    @cells = [" "," "," "," "," "," "," "," "," "]
+  end
+
+  def display
+    puts " #{cells[0]} | #{cells[1]} | #{cells[2]} "
+    puts "-----------"
+    puts " #{cells[3]} | #{cells[4]} | #{cells[5]} "
+    puts "-----------"
+    puts " #{cells[6]} | #{cells[7]} | #{cells[8]} "
+  end
+
+  def position(space)
+    input = (space.to_i - 1)
+    cells[input]
+  end
+
+  def full?
+    !@cells.include?(" ")
+  end
+
+  def turn_count
+    turn = 0
+    @cells.each {|cell| turn += 1 if cell == "X" || cell == "O"}
+    turn
+  end
+
+  def taken?(space)
+    input = space.to_i - 1
+    return true if cells[input] != " "
+    return false if cells[input] == " "
+  end
```

Comment:
Use Ruby's implicit return when returning booleans instead of `if` or `if`/`else` statements.  In this case, 
``` ruby
def taken?(space)
    !(position(space) == " " || position(space) == "")
end
```
where `#position` is a method that takes care of translating the index for us..